### DISCUSSION TITLE
23.02  install with new Maria DB version

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1421] [General]
I 've been asked to standardize to RHEL 8  I would like to use this opportunity bring LAMP to current versions.  My question now would be  Is it possible to perform a new installation of 23.02 using a new version of Maria DB  and migrate previous Db to the new ?

**Comment by wilpig** [comment]
Yes.  And I would do RHEL9 not 8 so ti will stay as a valid build for longer.
